guides/common/modules/proc_viewing-multiple-content-view-environment-activation-keys.adoc

```diff
@@ -0,0 +1,38 @@
+[id="Viewing_multiple_content_view_environment_activation_keys_{context}"]
+= Viewing multiple content view environment activation keys
+
+Use this procedure to view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+* Ensure you have activation keys created.
+For more information, see xref:Creating_an_Activation_Key_{context}[Creating an activation key].
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.      
+. Select an activation key.
+. Under the *Details* tab, the *Content view environments* are displayed.
+
+[NOTE]
+====
+You cannot reassign a multi-environment activation key's content view environments using the web UI.
+To change content view environments, update the activation key with Hammer.
+====
+
+.CLI procedure
+. Run the following command to view a list of activation keys in your organization including single environment and multi-environment keys:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key list --organization _“Default Organization”_
+----
+The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each activation key. The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment activation keys.
++
+. Run the following command to view multi-environment information for a single activation key:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key info --id _[1,2,3]_
```

Comment:
`--id` is just a single number, not an array.

I see elsewhere in the docs we do it like this:

```suggestion
$ hammer activation-key info --id _My_Activation_Key_ID_
```